lms/resources/_js_config/file_picker_config.py

```diff
@@ -52,6 +52,9 @@ def canvas_config(cls, request, application_instance):
         course_id = request.lti_params.get("custom_canvas_course_id")
         config = {
             "enabled": files_enabled,
+            "withFolders": application_instance.settings.get(
+                "canvas", "file_picker_folders"
+            ),
```

Comment:
Maybe `"foldersEnabled"` to match the naming of the `canvas.folders_enabled` or `file_picker_folders_enabled` application instance setting? (If the setting gets renamed as I suggested.)

```suggestion
            "foldersEnabled": application_instance.settings.get(
                "canvas", "folders_enabled"
            ),
```

Reply:
I'm wondering why the frontend code needs this setting? Can't the API responses just either contain folders or not and the frontend can react accordingly?